email_mailbox/src/components/EmailWrapper.js

```diff
@@ -123,6 +240,14 @@ class EmailWrapper extends Component {
     this.props.onEditDraft();
   };
 
+  handleContactIsTrusted = async (contact, contactIds) => {
+    if (contact.isTrusted) {
+      return contact.isTrusted === 1;
```

Comment:
why isn't `contact.isTrusted` already a boolean?

Reply:
not sure why, but database give me 0 and 1, and even i can use the 1 and 0's i prefer to be type explicit